app/models/employee.rb

```diff
@@ -60,15 +91,19 @@ def salary_on(date)
   end
 
   def salary_data
```

Comment:
It looks like `salary_data` and `ending_salary_data_hashes` better belong outside of the model. Currently the model knows too much about the data hash we end up needing.

Reply:
I think you're right, but I'm also not sure if there's a place where they do belong right now. The project doesn't really make use of helpers or anything like that as it is, and we have a lot of stuff crammed into the employee model that we only need for one of the views. There's enough of it that it should probably be its own PR though, especially since this one is already pretty big.

Reply:
These existed prior to this PR so I'm inclined to agree. Let's make and issue and leave it for now.